src/plugins/intel_cpu/src/transformations/snippets/aarch64/pass/snippets_mark_skipped.cpp

```diff
@@ -192,6 +192,28 @@ void MarkSubgraphOpAsSkipped(const std::shared_ptr<Node> &node) {
     }
 }
 
+bool isSuitableConvert(const std::shared_ptr<const Node>& node) {
+    if (!ov::is_type<ov::op::v0::Convert>(node))
+        return false;
+    auto isSuitableParent = [](const std::shared_ptr<const Node>& node) {
```

Comment:
Could you please elaborate why did you add these checks for Convert? What is the reason?

Reply:
I followed the behavior of GraphOptimizer here https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/graph_optimizer.cpp#L3013-L3065. Thanks Alexandra!

Reply:
Oh! Absolutely `Assign`!  Applied. Thanks Alexandra!